### PR TITLE
[NO GBP] Fixes my fuckups with species livers

### PR DESCRIPTION
**Pull request by ChungusGamer666**
## About The Pull Request

Fixes https://github.com/tgstation/tgstation/issues/76329
I DID A REAL STUPID MISTAKE WHILE CODING https://github.com/tgstation/tgstation/pull/76184 I AM SORRY
The signal was sending the fucking human instead of seconds_per_tick

## Why It's Good For The Game

Fixes a BUNCH of liver behavior including plasmamen livers not healing wounds

## Changelog

:cl:
fix: Plasma will now heal plasmamen properly
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/_species.dm [hunk 1]
```diff
@@ -515,8 +515,6 @@ GLOBAL_LIST_EMPTY(features_by_species)
  */
 /datum/species/proc/on_species_loss(mob/living/carbon/human/C, datum/species/new_species, pref_load)
 	SHOULD_CALL_PARENT(TRUE)
-	if(C.dna.species.exotic_bloodtype)
-		C.dna.blood_type = random_blood_type()
```
Comment (ChungusGamer666) [inline code/modules/mob/living/carbon/human/_species.dm:519]: this check is redundant and is actually on on_species_gain, before you ask

**Comment by ChungusGamer666** [comment]
@Time-Green merge this asap i am such a fuckcing idiot :)

**Comment by MrMelbert** [comment]
Now's looking like a great time to make some species unit tests

**Comment by ChungusGamer666** [comment]
> Now's looking like a great time to make some species unit tests

Will do so in the future.

**Comment by ZephyrTFA** [comment]
banning you from the repo for lack of bitches

**Comment by ChungusGamer666** [comment]
> banning you from the repo for lack of bitches

image

**Comment by san7890** [comment]
two maintainer commented but zero reviews. sad!

**Review by san7890 — APPROVED** [review]
this stupid automerge button has been greyed out for three minutes

**Comment by ChungusGamer666** [comment]
this is so sad

**Pull request merged**